cmd/res/configuration.toml

```diff
@@ -6,6 +6,14 @@ LogLevel = "INFO"
       [Writable.InsecureSecrets.DB.Secrets]
       username = ""
       password = ""
+  [Writable.Telemetry]
+  Interval = "30s"
+  PublishTopicPrefix  = "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
+    [Writable.Telemetry.Metrics] # All service's metric names must be present in this list.
+    MessagesReceived = false
+    PipelineMessagesProcessed = false
+    PipelineMessageProcessingTime = false
+    [Writable.Telemetry.Tags]
```

Comment:
Forgot that the Device SDK doesn't support Metric yet, so we should drop this section for now. Sorry.